.travis.yml

```diff
@@ -108,7 +108,7 @@ jobs:
         - npm -v
       script:
         - echo "$DOCKER_PASSWORD" | docker login -u "$DOCKER_USERNAME" --password-stdin;
-        - ./ciBuildDocker.sh --version=${BUILD_VERSION} --docker-publish-type=publishLocal --add-dev-model=$ADD_DEV_MODEL --docker-update-latest=false
+        - ./ciBuildDocker.sh --version=${BUILD_VERSION} --docker-publish-type=publishLocal --add-dev-model=$ADD_DEV_MODEL
```

Comment:
`--docker-update-latest` is by default set to `true` in `ciBuildDocker.sh`.
Do we need to tag this image as latest locally (I wonder because it is not published as `latest` outside in this scenario)?

Reply:
Good catch! I've removed default from there, added some comment and better protected from passing invalid value in `.travis.yml`. Also pushed the same HEAD to preview branch to make sure if "staging docker builds" pipeline is still ok after this change: https://travis-ci.org/github/TouK/nussknacker/builds/684327262. Thank you for the review!